projects/m1/002-bottle-deposits/js/index.js

```diff
@@ -0,0 +1,22 @@
+//calcolo numero contenitori totali prelevati dall'utente
+let bottleOneL = 2;
+let bottlePlusOneL = 5;
+let cashRefoundOneL = 0.10;
```

Comment:
It is recommended to see:
https://www.freecodecamp.org/news/var-let-and-const-whats-the-difference/